policyengine_us/variables/gov/states/ne/tax/income/credits/cdcc/ne_cdcc_nonrefundable.py

```diff
@@ -17,7 +17,7 @@ def formula(tax_unit, period, parameters):
         p = parameters(period).gov.states.ne.tax.income.credits
         # determine AGI eligibility
         us_agi = tax_unit("adjusted_gross_income", period)
-        agi_eligible = us_agi > p.cdcc.agi_threshold
+        agi_eligible = us_agi > p.cdcc.refundable.income_threshold
```

Comment:
Should be >=, but this is cleaner (remove agi variable)
```suggestion
        eligible = ~tax_unit("ne_cdcc_refundable_eligible", period)
```